Classes/Hook/FileListButtonListener.php

```diff
@@ -77,7 +77,7 @@ public function addButton(ModifyButtonBarEvent $event): void
      */
     private function getButtons(array $buttons): array
     {
-        if (!$this->isFileListModuleUri()) {
+        if (!$this->isFileListModuleUri() || !$this->isTargetIdSet()) {
```

Comment:
There is a fallback builtin already, yet it seems a fallback missing. How about adding the fallback here: https://github.com/programmfabrik/typo3-easydb-plugin/pull/23/files#diff-c0f4b7f02a22454977593093063f00c795825aee354e5f356fad95456cf63cf2R211

```php
$folderId = $GLOBALS['TYPO3_REQUEST']?->getQueryParams()['id'] ?? $_GET['id'] ?? null;
```